### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by vaughnwalters**
first submission of js spa quiz.

### CODE REVIEW

File: events.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          targetDiv = clickEvent.currentTarget;
+          events.resetValues(targetDiv);
+          events.increaseBorder(targetDiv, "gray");
+          events.clearInput();
+        });
+      };   
+
+// add Event listener for keyup
+      input.addEventListener("keyup", function (keyupEvent){
+        events.changeDescription(targetDiv, keyupEvent);
```
Comment (JoeShep) [inline events.js:30]: You'll need some kind of check here to make sure a card has been selected. As it is, users can type into the input and it throws a console error "Cannot read property 'id' of undefined".

File: events.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+var CarLot = (function (events) {
+  var targetDiv;
+  var cards = document.getElementsByClassName("newDiv");
+  var input = document.getElementById("inputBox");
+    events.activateEvents = function() {
```
Comment (JoeShep) [inline events.js]: unnecessary indentation

**Comment by JoeShep** [comment]
Vaughn, this looks great.  Only a couple of nitpicks noted in the code. I prefer as much CSS as possible done in the CSS itself and not added with JS (except the border color, of course), but I can live with the `.style` method for now. Fix the validation problem when users type before a card is selected and you're good to merge.

**Comment by vaughnwalters** [comment]
Okay Joe, I just fixed the validation problem with an if statement that throws an alert when the field is typed in before a card is selected, and merged the quiz branch to master.

**Comment by JoeShep** [comment]
👍

**Pull request merged**